### ISSUE TITLE
[Bug] Plugin broken on latest Discord version

### ISSUE THREAD

**Issue by melon095** [Yentis/betterdiscord-emotereplacer#84]
```
EmoteReplacer Error in onSendMessage TypeError: Cannot read properties of undefined (reading 'split')
    at SendMessageService.getTargetEmoteFromMessage (EmoteReplacer.plugin.js:2974:27)
    at SendMessageService.onSendMessage (EmoteReplacer.plugin.js:2822:34)
    at Object.eval [as callback] (EmoteReplacer.plugin.js:3486:33)
    at Object.sendMessage (renderer.js:5:22871)
    at 64376ca0f49bef4402d8.js:1:164650
```

## Discord Info
stable 289103 (6f1a424) Host 1.0.9143 x64 (46952) Windows 10 64-bit (10.0.19043)

## BetterDiscord
stable 1.10.1

**Comment by OshidaBCF** [comment]
when does discord doesn't break anything

**Comment by Yentis** [comment]
Fixed in 2.1.6
